src/lib/database/entities/StarboardEntity.ts

```diff
@@ -248,41 +226,76 @@ export class StarboardEntity extends BaseEntity {
 		this.stars = this.#users.size;
 	}
 
+	/**
+	 * The embed for the message
+	 */
+	private getEmbed(language: Language) {
+		if (this.#starMessage?.embeds.length) {
+			return this.#starMessage.embeds[0].setColor(this.color);
+		}
+
+		const message = this.#message;
+		return new MessageEmbed()
+			.setAuthor(message.author.username, message.author.displayAvatarURL({ size: 128, format: 'png', dynamic: true }))
+			.setColor(this.color)
+			.setDescription(this.getContent(language))
+			.setTimestamp(message.createdAt)
+			.setImage(getImage(message)!);
+	}
+
+	/**
+	 * The text
+	 */
+	private getContent(language: Language) {
+		const url = `[${language.get(LanguageKeys.Misc.JumpTo)}](${this.#message.url})`;
```

Comment:
Honestly, this pattern should be done in the translation string (so you end up with `language.get(JumpTo, { url })`